cdap-api/src/main/java/co/cask/cdap/api/service/ServiceWorkerContext.java

```diff
@@ -41,6 +41,12 @@
   void execute(TxRunnable runnable);
 
   /**
+   * Execute a set of operations on datasets via a {@link TxRunnable} that are committed as a single transaction.
+   * @param runnable The runnable to be executed in the transaction
+   */
+  void execute(co.cask.cdap.api.service.TxRunnable runnable);
```

Comment:
You have to mark this method as deprecate since this method will get removed in future version.